### PR TITLE
Image Loading

### PR DESCRIPTION
**Pull request by jeffsyang**
@0x00b1 @bethac07 @AetherUnbound

### CODE REVIEW

File: cellprofiler/gui/pathlist.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def __init__(self, *args, **kwargs):
             self.SetFont(self.DROP_FILES_AND_FOLDERS_FONT)
             self.drop_files_and_folders_text_extent = self.GetFullTextExtent(
                 self.DROP_FILES_AND_FOLDERS_HERE
-            )
+            )[:2]
```
Comment (AetherUnbound) [inline cellprofiler/gui/pathlist.py]: Did the return type for this change? This seems suspect

File: cellprofiler/gui/pipelinecontroller.py [hunk 2]
```diff
@@ -2112,36 +2112,30 @@ def add_paths_to_pathlist(self, filenames):
...
+                dlg.SetSize(min_width, h)
+
             queue = six.moves.queue.Queue()
             interrupt = [False]
             message = ["Initializing"]
 
             def fn(filenames=filenames, interrupt=None, message=None, queue=queue):
-                if message is None:
-                    message = message
-                if interrupt is None:
```
Comment (AetherUnbound) [inline cellprofiler/gui/pipelinecontroller.py]: Are we asserting that interrupt will now exist? It seems possible for it to be `None` in which case `interrupt[0]` will fail.

File: cellprofiler/gui/pathlist.py (hunk 1 above)
Comment (jeffsyang) [reply cellprofiler/gui/pathlist.py]: tbh, i'm not entirely sure how to interpret the documentation at https://wxpython.org/Phoenix/docs/html/wx.GraphicsContext.html#wx.GraphicsContext.GetFullTextExtent but after stepping through the code, it returned 4 values, hence the splice. Please let me know if a better method comes to mind.

File: cellprofiler/gui/pipelinecontroller.py (hunk 2 above)
Comment (jeffsyang) [reply cellprofiler/gui/pipelinecontroller.py]: hm... i might have made a mistake here; i'll need to run some tests and figure out the details regarding threading and sharing variables.

**Pull request merged**